src/proto/beats.proto

```diff
@@ -10,10 +11,23 @@ enum Source {
   USER = 2;
 }
 
+enum Type {
+  BEAT = 0;
+  BAR = 1;
+  PHRASE = 2;
+}
+
+message Signature {
```

Comment:
rename this to TimeSignature. "Signature" without context can have many meanings

Reply:
Being in the context of of a group of Beats, I will think that is clear enough.
But we are not going to argue on that.
On the other hand. You're showing the protobuff. But I will assume that you also mean the wrapper class.
Done